Resources/config/imagine_vips.xml

```diff
@@ -0,0 +1,15 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<container xmlns="http://symfony.com/schema/dic/services"
+    xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+    xsi:schemaLocation="http://symfony.com/schema/dic/services http://symfony.com/schema/dic/services/services-1.0.xsd">
+
+    <services>
+
+        <service id="liip_imagine.vips" class="Imagine\Vips\Imagine" public="false">
```

Comment:
are these always autoregistered or should we add tags to the service so it always works?

Reply:
ah sorry, found it. the name is used in https://github.com/liip/LiipImagineBundle/blob/ee1581327e4de483e3359f6bb6dcfe7415bf3042/DependencyInjection/LiipImagineExtension.php#L104 so this is all we need.